### ISSUE TITLE
[Suggestion] Don't autorefresh during bosses

### ISSUE THREAD

**Issue by meekmeekmeek** [wchill/steamSummerMinigame#104]
I know bosses are fast and it's not always going to be able to avoid them, but it's heartbreaking watching a boss get to 25 stacks of rain, health only slowly inching down, and then have the refresh kick in and stock clicking him for several seconds.

**Comment by luanmuniz** [comment]
Will that work?

```javascript
function autoRefreshPage(autoRefreshMinutes) {
    refreshTimer = setTimeout(function() {
        autoRefreshHandler();
    }, autoRefreshMinutes * 1000 * 60);
}

function autoRefreshHandler() {
     if(!targetIsBoss) {
        w.location.reload(true);
    } else {
        setTimeout(function() {
                autoRefreshHandler();
        }, 3000)
    }
}
```

**Comment by oedw** [comment]
Working on it atm, 

@luanmuniz 
something like this:

```javascript
function autoRefreshHandler() {
     advLog('Target type = ' + s().m_rgPlayerData.target , 5);
	 if(s().m_rgPlayerData.target != ENEMY_TYPE.BOSS) {
        advLog('Refreshing, not boss', 5);
		w.location.reload(true);
    } else {
		advLog('A boss!', 5);
        setTimeout(function() {
			autoRefreshHandler();
        }, 3000);
    }
}
```

**Comment by oedw** [comment]
Welp! just realized target is not type, gonna fix that

**Comment by oedw** [comment]
Working on it atm, 

@luanmuniz 
something like this:

```javascript
function autoRefreshHandler() {
     advLog('Target type = ' + s().m_rgPlayerData.target , 5);
	 if(s().GetEnemy(s().m_rgPlayerData.current_lane, s().m_rgPlayerData.target).m_data.type != ENEMY_TYPE.BOSS) {
        advLog('Refreshing, not boss', 5);
		w.location.reload(true);
    } else {
		advLog('A boss!', 5);
        setTimeout(function() {
			autoRefreshHandler();
        }, 3000);
    }
}
```

**Comment by luanmuniz** [comment]
Should we send a Pull Request here or at https://github.com/SteamDatabase/steamSummerMinigame??

**Comment by wchill** [comment]
Solution available, needs testing #121
